PersonenOrt.Repository.Service/Controllers/PersonController.cs

```diff
@@ -25,9 +25,25 @@ public IEnumerable<Person> Get()
             }
         }
         [HttpPut("{id:int}")]
-        public Person PutPerson(int id, Person person)
+        public IActionResult PutPerson(int id, Person person)
```

Comment:
Das ist Spannend! In "meiner" Verison war das schon so drinnen... aber bist wohl von einen anderen Branch ausgegangen...